src/components/Survey/Survey.tsx

```diff
@@ -5,26 +5,13 @@ import './Survey.css'
 import bkg_img_2 from '../../images/survey/Big Shoes - Sitting on Rock Blue.png'
 import bkg_img_1 from '../../images/survey/Big Shoes - Sitting on Rock Yellow.png'
 export const Survey: React.FC = () => {
-  const thisContext = useContext(QuestionContext)
-  const [answerKey, updateAnswers] = useState<AllQuestionFormat | null>(null);
-  let questionKeys: null;
-  useEffect(() => {
-    if(thisContext){
-    questionKeys = Object.keys(thisContext).reduce((acc: any,cur)=>{
-        acc[cur] = ''
-        console.log(acc)
-        return acc
-      },{})
-    updateAnswers(questionKeys)
-    }
-  }, [thisContext]);
   
   return (
-    <AnswerContext.Provider value={answerKey}>
       <section className='survey-container'>
-
+        {/*<Question currentQuestion={currentQuestion} updateAllAnswers={updateAllAnswers}/> */}
         <div className='inner-container'>
```

Comment:
updateAllAnswers is how we're updating context?

Reply:
Yeah! this is a bit out of sorts as everything is commented out until we implement routing.

`updateAllAnswers` is the essentially just the `setState` of the hook, which on this line is passed down to the question component

Currently I am working on some ideas for routing, and potentially moving AnswerContext up to App for the sake of simplicity